__tests__/src/rules/accessible-emoji-test.js

```diff
@@ -45,6 +45,10 @@ ruleTester.run('accessible-emoji', rule, {
     },
   )).map(parserOptionsMapper),
   invalid: parsers.all([].concat(
+    {
+      code: '<Box as="input" type="hidden">🐼</Box>',
+      settings: { 'jsx-a11y': { polymorphicPropName: 'as' } },
+    },
```

Comment:
maybe this was meant to be a valid test?
```suggestion
    {
      code: '<Box as="input" type="hidden">🐼</Box>',
      settings: { 'jsx-a11y': { polymorphicPropName: 'as' } },
    },
  )).map(parserOptionsMapper),
  invalid: parsers.all([].concat(
```

Reply:
Yikes, Sorry I ran these locally but I think the merging messed me up 😞

Reply:
np, it happens :-) this might be ready to land after things pass!